src/errors/runtime/templates.js

```diff
@@ -6,43 +6,43 @@
+    [RUNTIME_ERRORS.portsOptionRequiresTwoNumbers]:                                    'The "--ports" option requires two numbers to be specified.',
+    [RUNTIME_ERRORS.portIsNotFree]:                                                    'The specified {portNum} port is already in use by another program.',
+    [RUNTIME_ERRORS.invalidHostname]:                                                  'The specified "{hostname}" hostname cannot be resolved to the current machine.',
+    [RUNTIME_ERRORS.cannotFindSpecifiedTestSource]:                                    'Cannot find a test source file at "{path}".',
+    [RUNTIME_ERRORS.clientFunctionCodeIsNotAFunction]:                                 '{#instantiationCallsiteName} code is expected to be specified as a function, but {type} was passed.',
+    [RUNTIME_ERRORS.selectorInitializedWithWrongType]:                                 '{#instantiationCallsiteName} is expected to be initialized with a function, CSS selector string, another Selector, node snapshot or a Promise returned by a Selector, but {type} was passed.',
+    [RUNTIME_ERRORS.clientFunctionCannotResolveTestRun]:                               "{#instantiationCallsiteName} cannot implicitly resolve the test run in context of which it should be executed. If you need to call {#instantiationCallsiteName} from the Node.js API callback, pass the test controller manually via {#instantiationCallsiteName}'s `.with({ boundTestRun: t })` method first. Note that you cannot execute {#instantiationCallsiteName} outside the test code.",
+    [RUNTIME_ERRORS.regeneratorInClientFunctionCode]:                                  `{#instantiationCallsiteName} code, arguments or dependencies cannot contain generators or "async/await" syntax (use Promises instead).`,
+    [RUNTIME_ERRORS.invalidClientFunctionTestRunBinding]:                              'The "boundTestRun" option value is expected to be a test controller.',
+    [RUNTIME_ERRORS.invalidValueType]:                                                 '{smthg} is expected to be a {type}, but it was {actual}.',
+    [RUNTIME_ERRORS.unsupportedUrlProtocol]:                                           'The specified "{url}" test page URL uses an unsupported {protocol}:// protocol. Only relative URLs or absolute URLs with http://, https:// and file:// protocols are supported.',
+    [RUNTIME_ERRORS.testControllerProxyCannotResolveTestRun]:                          `Cannot implicitly resolve the test run in the context of which the test controller action should be executed. Use test function's 't' argument instead.`,
+    [RUNTIME_ERRORS.timeLimitedPromiseTimeoutExpired]:                                 'Timeout expired for a time limited promise',
+    [RUNTIME_ERRORS.cannotUseScreenshotPathPatternWithoutBaseScreenshotPathSpecified]: 'Cannot use the screenshot path pattern without a base screenshot path specified.',
```

Comment:
Why do we revert to an old error message here?

Reply:
@AndreyBelym